### PR TITLE
dtest for 8989 & 9502

### PR DESCRIPTION
**Pull request by snazy**
dtest for "Reading from table which contains collection type using token function and with CL > ONE causes overwhelming writes to replicas" (CASSANDRA-8989)

### CODE REVIEW

File: read_repair_test.py [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+from cassandra import ConsistencyLevel
+from cassandra.query import SimpleStatement
+
+from dtest import Tester, PRINT_DEBUG
+from tools import since
+
+
+@since('2.0')
+#@require('CASSANDRA-9502')
```
Comment (ptnapoleon) [inline read_repair_test.py]: Remove the commented out line, please.

File: read_repair_test.py [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+        future = cursor1.execute_async(query, trace=True)
+        future.result()
+        trace = future.get_query_trace(max_wait=120)
+        self.pprint_trace(trace)
+        for trace_event in trace.events:
+            # Step 1, find coordinator node:
+            activity = trace_event.description
+            assert activity != "Appending to commitlog"
+            assert activity != "Adding to cf memtable"
+            assert activity != "Acquiring switchLock read lock"
```
Comment (ptnapoleon) [inline read_repair_test.py]: Could you make these perform substring searches, instead of using `!=`?

File: read_repair_test.py [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+from dtest import Tester, PRINT_DEBUG
+from tools import since
+
+
+@since('2.0')
+#@require('CASSANDRA-9502')
+class TestRepairDataSystemTable(Tester):
+    """
+    @jira_ticket CASSANDRA-8989
+    @jira_ticket CASSANDRA-9502
```
Comment (ptnapoleon) [inline read_repair_test.py]: Thanks for the doxygen markup!

Comment (snazy) [inline read_repair_test.py]: copy + paste ;)

File: read_repair_test.py (hunk 2 above)
Comment (snazy) [inline read_repair_test.py]: is it ok? (python question ;) )

Comment (ptnapoleon) [inline read_repair_test.py]: Is the substring search okay? The best way to do it would be `assert "Appending to commitlog" not in activity`

**Comment by ptnapoleon** [comment]
Thanks. +1

**Pull request merged**